db/seeds/development/organisation_admins.seeds.rb

```diff
@@ -0,0 +1,17 @@
+after 'development:organisations', 'development:school_admins' do
+  puts 'Seeding organisation_admins'
+
+  School.all.each do |school|
+    admin =
+      school
+        .school_admins
+        .joins(:user)
+        .where(users: { email: 'admin@example.com' })
+        .first
+
+    school.organisations.each do |organisation|
+      OrganisationAdmin.where(user: admin, organisation: organisation)
```

Comment:
Looks like this is redundant for school admins, as they by default have access to all organizations.

Reply:
Yeah, this was from the initial implementation where school admins didn't have direct access to these pages. Fixed.